independent-publisher-connectors/HubSpot CRM/apiProperties.json

```diff
@@ -0,0 +1,75 @@
+{
+  "properties": {
+    "connectionParameterSets": {
+      "uiDefinition": {
+        "displayName": "Authentication Type",
+        "description": "Type of authentication to be used."
+      },
+      "values": [
+        {
+          "name": "hubspot-oauth",
+          "uiDefinition": {
+            "displayName": "Use HubSpot OAuth 2.0",
+            "description": "Log in using HubSpot app client ID and secret."
+          },
+          "parameters": {
+            "token": {
+              "type": "oauthSetting",
+              "oAuthSettings": {
+                "identityProvider": "oauth2",
+                "clientId": "[Your_Value]",
+                "scopes": [
+                  "contacts oauth"

```

Comment:
The scopes here and in the swagger file don't match. Can you please update the correct scopes? ApiProperties is the one used in down stream application. Please update all HubSpot connector with similar changes.